buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -75,6 +75,7 @@ object Dependencies {
     const val slf4j_timber = "com.arcao:slf4j-timber:3.1@aar"
     const val emoji_java = "com.vdurmont:emoji-java:5.1.1"
     const val json_schema_validator = "com.networknt:json-schema-validator:1.0.66"
+    const val splashscreen = "androidx.core:core-splashscreen:1.0.0-beta02"
```

Comment:
None of us love bringing in beta dependencies but I agree it's the way to go here.